### PR TITLE
Add globalstyles and default theme

### LINKED ISSUE #13: Add global styles
https://styled-components.com/docs/basics#getting-started

### PR DESCRIPTION
**Pull request by Susanne1307**
Add emotion-theming:
https://emotion.sh/docs/emotion-theming

Use ThemeProvider in Storybook:
https://dev.to/ryanlanciaux/use-a-themeprovider-component-in-storybook-3m63

closes #13

### CODE REVIEW

File: client/src/GlobalStyles.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Global, css } from '@emotion/core';
+
+function GlobalStyles() {
+  return (
+    <Global
+      styles={(theme) => css`
+        @import url('https://fonts.googleapis.com/css2?family=Comfortaa:wght@300;400;500;600;700&display=swap');
```
Comment (hauptdigital) [inline client/src/GlobalStyles.js:8]: Good solution to move the font import to `GlobalStyles` as now all global styles can be found in one file. 👍

File: package.json [hunk 2]
```diff
@@ -50,6 +50,7 @@
   ],
   "dependencies": {
     "dotenv": "^8.2.0",
+    "emotion-theming": "^10.0.27",
```
Comment (hauptdigital) [inline package.json:53]: I Just a minor tweak. `emotion-theming` is only used in the client part of your application (= React). Maybe this dependency can be moved to `client/package.json`

File: client/src/components/FooterButton.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ export const FooterButton = styled.button`
   cursor: pointer;
   &:hover {
     color: #ffffff;
```
Comment (hauptdigital) [inline client/src/components/FooterButton.js:19]: Just a minor issue. In some components I can see that you have a mix of colors coming from your `theme.js` and some values are defined static in the component file (example `#ffffff`). If every color comes from the theme file, you make later changes in the colors of your app easier.

File: package.json (hunk 2 above)
Comment (Susanne1307) [reply package.json:53]: Done, thanks!

**Pull request merged**